CONTRIBUTING.md

```diff
@@ -72,136 +85,44 @@ To recap, when adding new tasks or states to Porter's CloudFormation template:
 
-## Internal Messaging
+### Internal Messaging
 
-How data is passed between the various states of the Step Function is key to building a reliable state machine, and understanding the flow of data in invaluable when developing this project. The following is a detailed look at the lifecycle of inputs and outputs of the state machine internals. The code examples are meant to demonstrate important structural aspects, and are not necessarily 100% accurate.
+How data is passed into and between between the various states of the Step Function is key to building a reliable state machine, and understanding the flow of data in invaluable when developing this project. The following is a detailed look at the lifecycle of inputs and outputs of the state machine internals. The code examples are meant to demonstrate important structural aspects, and are not necessarily 100% accurate.
 
 [Input and output processsing](https://docs.aws.amazon.com/step-functions/latest/dg/concepts-input-output-filtering.html) is used extensively to ensure that the data being passed between states is clean, and the data being passed to task resources (Lambda functions, etc) is appropriately scoped. A good understanding of how the state I/O and task resource I/O interact is crucial.
 
-In some cases, states will use a result path of `$.Void` if their return values are unwanted. These are largely treated in the documentation below as though they don't exist.
-
-### Execution Input
-
-This is the input provided by the end user as raw JSON. The format is covered in detail in the [README](https://github.com/PRX/Porter/blob/master/README.md). `Job.Id` and `Job.Source` are consider required.
-
-```
-{ "Job": { "Id": "1234567890asdfghjkl", "Source": { "Mode": "AWS/S3" } } }
-```
-
-### Input Normalization
-
-Some states in the machine assume that certain properties exist in the input. In order for those properties to be optional for the user providing the input, the execution input it normalized as the first step.
-
-This state does no input or output processing at the machine level. It takes in the complete execution input, passes the entire input to the Lambda function, and uses the result of the Lambda function as the entire output result.
-
-- `InputPath`: `$`, `{ Job: { … } }`
```

Comment:
So these examples were all redundant with the README?

Reply:
No, these actually don't overlap at all with what's in the README. Those are all job message examples, i.e., from clients. These were data structures internal to the state machine definition itself. It was too hard to keep these docs in line with the code, and I decided this level of specificity was unnecessary. I generalized most of the information that mattered when thinking about inputs/outputs/processing in the rewritten docs